### ISSUE TITLE
Limit Terminal Connect Button Options in User Settings

### ISSUE THREAD

**Issue by thinkbig1979** [Ylianst/MeshCentral#5192]
**Is your feature request related to a problem? Please describe.**
Currently there is no way to force a user to log into the terminal with their own credentials without resorting to editing things like .bashrc, which has unwanted side-effects. Meshcentral however does offer a great option to right click on the Connect button and select "Login Shell", which opens the terminal with a login prompt rather than as the agent's default user (root in my case).

**Describe the solution you'd like**
It would be great if, in the user settings, you could limit what the 'Connect' button for the terminal connection does. I would like to be able to limit it to "Login Shell", so that that becomes the default behavior for that user, and right clicking on the Connect button no longer shows any other options.

**Describe alternatives you've considered**
Editing .bashrc has come up as an option, but that affects all users logging in, which is not desirable.

**Comment by si458** [comment]
very similar to #4558 and #1142

**Comment by thinkbig1979** [comment]
> very similar to #4558 and #1142

Indeed, both relate to being able to make adjustments to what the connect buttons do. Different use cases though.

**Comment by razdavidovich** [comment]
I am joining this request, and would like to explain the logic behind it for me.
We have some automation scrips available on a specific user home folder, and would like to change the default behavior of the button to the login shell.
![image](https://github.com/Ylianst/MeshCentral/assets/1929135/69c7b1ff-e869-4da4-a7c3-8dd11ec0ef45)

My main problem is that when using the mobile web site, there is no way to simulate a right click and selecting this menu.
So instead, if there is a way to control the default behavior, I can change it to Login Shell.
This is relevant to Linux, but maybe have an impact on other OS's

**Comment by razdavidovich** [comment]
@Ylianst is there a way to allow that from the mobile version?
